src/components/CompareResults/beta/RevisionHeader.tsx

```diff
@@ -38,10 +42,36 @@ const styles = {
   }),
 };
 
-function createTitle(header: RevisionsHeader) {
-  return header.test === '' || header.suite === header.test
-    ? header.suite
-    : `${header.suite} ${header.test}`;
+function createTitle(
+  header: RevisionsHeader,
+  docsURL: string,
+  isLinkSupported: boolean,
+) {
+  const isTestUnavailable = header.test === '' || header.suite === header.test;
+  if (isLinkSupported) {
+    return isTestUnavailable ? (
+      <Link
+        aria-label='link to suite documentation'
+        underline='hover'
+        href={docsURL}
+      >
+        {header.suite}{' '}
```

Comment:
The space here is underlined as well while hovering. Could you move `{' '}` outside of the `Link` component.